py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -483,7 +488,11 @@ def element_attribute_to_include(locator: Tuple[str, str], attribute_: str) -> C
 
     def _predicate(driver: WebDriverOrWebElement):
         try:
-            element_attribute = driver.find_element(*locator).get_attribute(attribute_)
+            element_attribute = (
+                driver.find_element(*locator).get_dom_attribute(attribute_)
+                if attribute_ not in attr_get_property
+                else driver.find_element(*locator).get_property(attribute_)
+            )
```

Comment:
You might need to translate the names on the caller site if one of PROPERTY_ALIASES (defined in the /attribute.js) is used.
https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/webdriver/atoms/attribute.js